docs/tutorials/building-a-new-site-wordpress-and-gatsby.md

```diff
@@ -357,10 +357,10 @@ You can apply the same procedure to calling and creating pages, custom post type
 
 # Up Next :point_right:
 
-- :boat: [Migrating from other WP source plugins](./migrating-from-other-wp-source-plugins.md)
+- :boat: [Migrating from other WP source plugins](../migrating-from-other-wp-source-plugins.md)
 - :house: [Hosting WordPress](./hosting.md)
```

Comment:
Thanks for noticing this and opening a PR! It looks like this one might need to be updated too.

Reply:
Oh! Thanks!!
I fixed it. https://github.com/gatsbyjs/gatsby-source-wordpress-experimental/pull/191/commits/3992604cd2e5c523c5c82a3466639cb880bfeb66